src/mobile/components/avant_garde_tabs/index.styl

```diff
@@ -18,7 +18,7 @@
   position absolute
   left 0
   top 0
-  z-index 1000
+  z-index 1
```

Comment:
Why was this needed?

Reply:
The mobile fairs nav appears on top of the expanded nav:

<img width="448" alt="Screen Shot 2019-05-13 at 7 53 46 PM" src="https://user-images.githubusercontent.com/236943/57667589-f49ef080-75b8-11e9-80c0-8e535a3c3a20.png">